packages/camera/camera/example/lib/main.dart

```diff
@@ -132,7 +131,7 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
     } else {
       return AspectRatio(
         aspectRatio: controller.value.aspectRatio,
-        child: CameraPreview(controller),
+        child: controller.buildPreview(),
```

Comment:
If the CameraPreview widget will be just pumping frames from the camera to the app UI, it's OK to remove the CameraPreview widget, HOWEVER, I still think that a nicely implemented CameraPreview widget (for example something that has a shutter button, and ISO/aperture settings... or filters, or whatever) is good!

Why remove the CameraPreview widget?

Reply:
At this moment the `CameraPreview` only wraps the `Texture` widget (or if CameraId is null it will return an empty `Container`). 

I completely agree with you that a nice widget that can be used in place to access all basic features would be really helpful but at the moment we are far away from that. 

So wouldn't it be better to break the API now and force developers to rely on the `buildPreview` method (by removing the current widget, that basically does the same) and when things settle down introduce an additional widget for developers that want an out of the box solution?

Reply:
> removing the current widget, that basically does the same) and when things settle down introduce an additional widget for developers that want an out of the box solution?

I think we already have the widget, so we can add features later via props to the CameraPreview interface, and developers only need to "opt in", imagine something like:

`CameraPreview(controller, withShutterButton: true);`

It's good that the controller exposes the raw method in case advanced users want to create their own preview widget, but I don't think we should force everybody to use the lowest-level API available. WDYT?

Reply:
Yes that is indeed also a nice option, I will put back the `CameraPreview` widget.

Reply:
Thanks for adding this back!